### PR TITLE
Added multiple plugins

### PR DESCRIPTION
**Pull request by backnight**
- Added env var WORDPRESS_FORCE_SSL_ADMIN in order to be able to configure it to false in development mode (without SSL)
- Renamed WORDPRESS_DATABASE_HOST value because it was not matching with the service name in docker-compose
- Added custom Dockerfile in order to zip the local Wordpress plugins and to install them from the env configuration + in order to allow Edward to add custom Nginx plugins if necessary
- Added a custom "Elementor-GraphQL" Wordpress plugin in order to output the json configuration of Elementor into the GraphQL endpoint (processed by the NuxtJS frontend)
- Added a custom "Linkurious Elementor Widgets" Wordpress plugin in order to show the ability of creating custom and complex elements in Elementor editor that will be hot loaded by the frontend ("linkurious-test-slider" element as proof of concept with a repeatable list of images for a slider)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,3 +9,4 @@ WORDPRESS_S3_UPLOADS_BUCKET=lk-dev-wordpress
 WORDPRESS_S3_UPLOADS_REGION=fr-par
 WORDPRESS_S3_UPLOADS_KEY=key
 WORDPRESS_S3_UPLOADS_SECRET=secret
+WORDPRESS_FORCE_SSL_ADMIN=true
```
Comment (enys) [inline .env.example:12]: Self-note : This needs to be aded to CAC

File: plugins/elementor-graphql/elementor-graphql.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?php
+/*
+Plugin Name: Elementor Graphql
+Plugin URI: http://linkurio.us
+description: Output Elementor json data in graphql
+Version: 1
+Author: Agily x Linkurious
+Author URI: https://www.agily.fr
+License: GPL2
```
Comment (enys) [inline plugins/elementor-graphql/elementor-graphql.php:9]: @backnight A specific reason for GPL2 ?
We usually go with Apache 2

File: .env.example (hunk 1 above)
Comment (backnight) [reply .env.example:12]: CAC ?

File: plugins/elementor-graphql/elementor-graphql.php (hunk 2 above)
Comment (backnight) [reply plugins/elementor-graphql/elementor-graphql.php:9]: No specific reason, put the Licence you want.

File: .env.example (hunk 1 above)
Comment (enys) [reply .env.example:12]: Configuration As Code, an internal repository handling the ansible deployements. I need to reflect this added Env var there before merging this.

File: docker-compose.yml [hunk 3]
```diff
@@ -3,12 +3,10 @@ version: '3.9'
 
 services:
   wordpress-server:
-    image: bitnami/wordpress-nginx:${WORDPRESS_VERSION}
+    image: ${REPOSITORY}linkurious/docker-wordpress:${LINKURIOUS_WORDPRESS_VERSION:-0.0.1}
```
Comment (enys) [inline docker-compose.yml:6]: @backnight I am squashing your change here, as this would have an impact on our deployement

**Review by enys — APPROVED** [review]
I need to push our deployment repository before we merge this, but the rest is ok

**Pull request merged**